cerberus/cerberus.py

```diff
@@ -701,6 +722,13 @@ def _validate_allof(self, allof, field, value):
         if not valid:
             self._error(field, errorstack)
 
+    def _validate_rename(self, rename, field, value):
+        if isinstance(rename, _str_type):
```

Comment:
i don't get your point.
what hinders you from checking if the provided value in the schema is a string in `validate_schema`?